aslprep/interfaces/confounds.py

```diff
@@ -383,58 +395,45 @@ def _run_interface(self, runtime):
-                'Description': 'QEI calculated on mean SCORE-denoised CBF image.',
+            'qei_cbf_rho_ss': {
+                'LongName': 'Cerebral Blood Flow Quality Evaluation Index Rho SS',
+                'Description': 'QEI rho SS calculated on mean CBF image.',
                 'Term URL': 'http://indexsmart.mirasmart.com/ISMRM2017/PDFfiles/0682.html',
             },
-            'qei_cbf_scrub': {
-                'LongName': 'SCRUB-Denoised Cerebral Blood Flow Quality Evaluation Index',
-                'Description': 'QEI calculated on mean SCRUB-denoised CBF image.',
+            'qei_cbf_di': {
+                'LongName': 'Cerebral Blood Flow Quality Evaluation Index DI',
+                'Description': 'QEI DI calculated on mean CBF image.',
                 'Term URL': 'http://indexsmart.mirasmart.com/ISMRM2017/PDFfiles/0682.html',
             },
-            'qei_cbf_basil': {
-                'LongName': 'BASIL Cerebral Blood Flow Quality Evaluation Index',
-                'Description': 'QEI calculated on CBF image produced by BASIL.',
-                'Term URL': 'http://indexsmart.mirasmart.com/ISMRM2017/PDFfiles/0682.html',
-            },
-            'qei_cbf_basil_gm': {
-                'LongName': (
-                    'BASIL Partial Volume Corrected Cerebral Blood Flow Quality Evaluation Index'
-                ),
-                'Description': (
-                    'QEI calculated on partial volume-corrected CBF image produced by BASIL.'
-                ),
-                'Term URL': 'http://indexsmart.mirasmart.com/ISMRM2017/PDFfiles/0682.html',
-            },
-            'mean_gm_cbf': {
-                'LongName': 'Mean Cerebral Blood Flow of Gray Matter',
-                'Description': 'Mean CBF value of gray matter.',
-                'Units': 'mL/100 g/min',
-            },
-            'mean_wm_cbf': {
-                'LongName': 'Mean Cerebral Blood Flow of White Matter',
-                'Description': 'Mean CBF value of white matter.',
-                'Units': 'mL/100 g/min',
-            },
-            'ratio_gm_wm_cbf': {
-                'LongName': 'Mean Gray Matter-White Matter Cerebral Blood Flow Ratio',
-                'Description': (
-                    'The ratio between the mean gray matter and mean white matter CBF values.'
-                ),
-            },
             'percentage_negative_cbf': {
                 'LongName': 'Percentage of Negative Cerebral Blood Flow Values',
                 'Description': (
                     'Percentage of negative CBF values, calculated on the mean CBF image.'
                 ),
                 'Units': 'percent',
             },
+            # SCORE QEI metrics
+            'qei_cbf_score': {
+                'LongName': 'SCORE-Denoised Cerebral Blood Flow Quality Evaluation Index',
+                'Description': 'QEI calculated on mean SCORE-denoised CBF image.',
```

Comment:
Is it the mean CBF image? Based on `_load_one_image` I'd guess it was the _first_ CBF image in the time series.

Reply:
```
        (inputnode, compute_qc_metrics, [
            ('name_source', 'name_source'),
            ('asl_mask', 'asl_mask'),
            ('mean_cbf', 'mean_cbf'),
            ('confounds_file', 'confounds_file'),
        ]),
```

It's the mean cbf when the workflow is built. I bet the first volume stuff is a leftover from early days